cmd/manager/main.go

```diff
@@ -51,6 +51,12 @@ func main() {
 		log.Fatalf("failed to get watch namespace: %v", err)
 	}
 
+	// Report that the operator is progressing towards a new version.
```

Comment:
Is this really what you are doing here? The function is doing way for than that.